qa/pull-tester/rpc-tests.sh

```diff
@@ -32,6 +32,7 @@ testScripts=(
     'reindex.py'
     'decodescript.py'
     'p2p-fullblocktest.py'
+    'zmq_test.py'
```

Comment:
I think we need to move the `zmq_test.py` to the `testScriptsExt` list otherwise travis cannot find the zmq python module. Maybe there is also a way to make travis support python/zmq.